test/lexbor/html/tokenizer.c

```diff
@@ -27,7 +27,7 @@ typedef struct {
 tkz_test_cmp_t;
 
 
-lxb_html_token_t *
+static lxb_html_token_t *
```

Comment:
No need to mix patches. If the patch is about Meson then there should not be these corrections. This is a separate patch.